README.md

```diff
@@ -183,6 +184,118 @@ docker.run('ubuntu', ['bash', '-c', 'uname -a'], process.stdout, function (err,
   console.log(data.StatusCode);
 });
 ```
+The following list of attributes for create_options and start_options are taken from: https://docs.docker.com/engine/reference/api/docker_remote_api_v1.19/ (Create a container).
```

Comment:
Pedro, 
Are you rejecting the entire PR? 
Can you elaborate on " It should be checked and the source." I don't understand it. 

The reason I did the PR is that I struggled to understand the function parameters since they are inconsistently presented in the readme.md at least for docker.run. 
I had to google for the data from different pages. I am documenting what I found and I am convinced this will help others. 
Cheers, 



- Philippe 



From: "Pedro Dias" <notifications@github.com> 
To: "apocas/dockerode" <dockerode@noreply.github.com> 
Cc: "phil" <phil@keremi.com> 
Sent: Friday, January 8, 2016 12:10:54 PM 
Subject: Re: [dockerode] Added doc on docker.run (#210) 



In README.md : 
> @@ -183,6 +184,118 @@ docker.run('ubuntu', ['bash', '-c', 'uname -a'], process.stdout, function (err,
>    console.log(data.StatusCode);
>  });
>  ```
> +The following list of attributes for create_options and start_options are taken from: https://docs.docker.com/engine/reference/api/docker_remote_api_v1.19/ (Create a container). 


This shouldn't be included in here. It should be checked and the source. 

— 
Reply to this email directly or view it on GitHub .